Project Source Code/WinFIM.NET Service/Service1.cs

```diff
@@ -111,10 +111,11 @@ private void OnTimer(object sender, ElapsedEventArgs args)
         protected override void OnStop()
         {
             string serviceStopMessage = Properties.Settings.Default.service_stop_message +
-                                        $": (UTC) {DateTime.UtcNow:yyyy/dd/MM hh:mm:ss tt}\n\n";
+                                        $": (UTC) {DateTime.UtcNow:
```

Comment:
Blew this change.  Let my fingers go on remote control
Reject if necessary.  I will fix properly.
Thanks.

Reply:
Thanks @Bobberty for submitting the PR! 
I'll wait for the updated patch for line 114...

Reply:
Fixed....